### PR TITLE
Don't pass null kwargs to Saturn

### PR DESCRIPTION
**Pull request by jsignell**
Closes#CU-cykj8m, #CU-cyq830

### CODE REVIEW

**Review by jameslamb — APPROVED** [review]
I like it!

File: dask_saturn/core.py [hunk 1]
```diff
@@ -66,7 +65,7 @@ class SaturnCluster(SpecCluster):
         If no size is provided, this will default to the size configured for Jupyter
         in your Saturn Cloud project.
     :param worker_is_spot: Flag to indicate if workers should be started on Spot Instances nodes.
-        Defaults to False.
+        Added in dask-saturn 0.1.0.
```
Comment (jsignell) [inline dask_saturn/core.py:68]: Sorry @jameslamb - one last thing, do you think this should have a Saturn version as well?

Comment (jameslamb) [reply dask_saturn/core.py:68]: oh yeah, this should include the Saturn version if it's easy for users to know what Saturn version they're running

**Comment by hhuuggoo** [comment]
Task linked: [CU-cykj8m [dask-saturn] dask-saturn should only send explicity set parameters. Not defaults.](https://app.clickup.com/t/cykj8m)

**Pull request merged**